src/components/dao/dao.js

```diff
@@ -316,7 +317,6 @@ export default class DaoComponent extends Component {
         if (addrError.isBad) {
             noContract.isBad = true;
             noDao.isBad = true;
-            this.setState({ proposed: -1, proposedInfo: {} });
```

Comment:
Why remove this line? Its important for two reasons:
- if there is no dao, then there is not creation proposal
- `setState` updates the react component